lib/bpk-component-chip/stories.js

```diff
@@ -146,11 +169,32 @@ class StatefulBpkDismissibleChipExample extends React.Component<
     };
   }
 
-  removeChip = chip => {
+  removeChip = (chip: 'flights' | 'hotels' | 'carHire' | 'trains') => {
```

Comment:
Changed this function to be the recommendation from flow regarding string literals:

https://flow.org/en/docs/types/literals/